cpp/src/utilities/cython.cpp

```diff
@@ -27,28 +28,159 @@ namespace cython {
 // Populates a graph_container_t with a pointer to a new graph object and sets
 // the meta-data accordingly.  The graph container owns the pointer and it is
 // assumed it will delete it on destruction.
-//
-// FIXME: Should local_* values be void* as well?
 void populate_graph_container(graph_container_t& graph_container,
-                              legacyGraphTypeEnum legacyType,
-                              raft::handle_t const& handle,
-                              void* offsets,
-                              void* indices,
+                              raft::handle_t& handle,
+                              void* src_vertices,
+                              void* dst_vertices,
                               void* weights,
-                              numberTypeEnum offsetType,
-                              numberTypeEnum indexType,
+                              void* vertex_partition_offsets,
+                              numberTypeEnum vertexType,
+                              numberTypeEnum edgeType,
                               numberTypeEnum weightType,
-                              int num_vertices,
-                              int num_edges,
-                              int* local_vertices,
-                              int* local_edges,
-                              int* local_offsets,
+                              int num_partition_edges,
+                              int num_global_vertices,
+                              int num_global_edges,
+                              int partition_row_size,  // pcols
+                              int partition_col_size,  // prows
                               bool transposed,
                               bool multi_gpu)
 {
   CUGRAPH_EXPECTS(graph_container.graph_ptr_type == graphTypeEnum::null,
                   "populate_graph_container() can only be called on an empty container.");
 
+  bool do_expensive_check{false};
+  bool hypergraph_partitioned{false};
+
+  raft::comms::comms_t const& communicator = handle.get_comms();
+  int const rank = communicator.get_rank();
+  int partition_row_rank = rank / partition_row_size;
+  int partition_col_rank  = rank % partition_row_size;
```

Comment:
```
      auto& row_comm = handle.get_subcomm(cugraph::partition_2d::key_naming_t().row_name());
      auto const row_comm_rank = row_comm.get_rank();
      auto const row_comm_size = row_comm.get_size();
      auto& col_comm = handle.get_subcomm(cugraph::partition_2d::key_naming_t().col_name());
      auto const col_comm_rank = col_comm.get_rank();
      auto const col_comm_size = col_comm.get_size();
```

Reply:
and I'd recommend using row_comm_size instead of partition_row_size for consistency. Inconsistency in naming here can be very confusing and can be a source of many bugs.

Reply:
Using the handle to retrieve the ranks in the next commit, and changed var names to `*_comm_size` for consistency too.